account_credit_control/models/credit_control_run.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2012-2017 Camptocamp SA
+# Copyright 2017 Okia SPRL (https://okia.be)
+# Copyright 2020 Manuel Calero - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class CreditControlRun(models.Model):
+    """ Credit Control run generate all credit control lines and reject """
+
+    _name = "credit.control.run"
+    _rec_name = "date"
+    _description = "Credit control line generator"
+
```

Comment:
Add this:
```suggestion
    _order = "date DESC"

```
Or cherry-pick this:
https://github.com/OCA/credit-control/pull/61/files#diff-dee1c1533bb5904fd05aa4b7c6685c5cR15